edb/pgsql/resolver/information_schema.py

```diff
@@ -0,0 +1,812 @@
+        ("ordering_routine_schema", "sql_identifier"),
+        ("ordering_routine_name", "sql_identifier"),
+        ("reference_type", "character_data"),
+        ("data_type", "character_data"),
+        ("character_maximum_length", "cardinal_number"),
+        ("character_octet_length", "cardinal_number"),
+        ("character_set_catalog", "sql_identifier"),
+        ("character_set_schema", "sql_identifier"),
+        ("character_set_name", "sql_identifier"),
+        ("collation_catalog", "sql_identifier"),
+        ("collation_schema", "sql_identifier"),
+        ("collation_name", "sql_identifier"),
+        ("numeric_precision", "cardinal_number"),
+        ("numeric_precision_radix", "cardinal_number"),
+        ("numeric_scale", "cardinal_number"),
+        ("datetime_precision", "cardinal_number"),
+        ("interval_type", "character_data"),
+        ("interval_precision", "cardinal_number"),
+        ("source_dtd_identifier", "sql_identifier"),
+        ("ref_dtd_identifier", "sql_identifier"),
+    ],
+    "user_mapping_options": [
+        ("authorization_identifier", "sql_identifier"),
+        ("foreign_server_catalog", "sql_identifier"),
+        ("foreign_server_name", "sql_identifier"),
+        ("option_name", "sql_identifier"),
+        ("option_value", "character_data"),
+    ],
+    "user_mappings": [
+        ("authorization_identifier", "sql_identifier"),
+        ("foreign_server_catalog", "sql_identifier"),
+        ("foreign_server_name", "sql_identifier"),
+    ],
+    "view_column_usage": [
+        ("view_catalog", "sql_identifier"),
+        ("view_schema", "sql_identifier"),
+        ("view_name", "sql_identifier"),
+        ("table_catalog", "sql_identifier"),
+        ("table_schema", "sql_identifier"),
+        ("table_name", "sql_identifier"),
+        ("column_name", "sql_identifier"),
+    ],
+    "view_routine_usage": [
+        ("table_catalog", "sql_identifier"),
+        ("table_schema", "sql_identifier"),
+        ("table_name", "sql_identifier"),
+        ("specific_catalog", "sql_identifier"),
+        ("specific_schema", "sql_identifier"),
+        ("specific_name", "sql_identifier"),
+    ],
+    "view_table_usage": [
+        ("view_catalog", "sql_identifier"),
+        ("view_schema", "sql_identifier"),
+        ("view_name", "sql_identifier"),
+        ("table_catalog", "sql_identifier"),
+        ("table_schema", "sql_identifier"),
+        ("table_name", "sql_identifier"),
+    ],
+    "views": [
+        ("table_catalog", "sql_identifier"),
+        ("table_schema", "sql_identifier"),
+        ("table_name", "sql_identifier"),
+        ("view_definition", "character_data"),
+        ("check_option", "character_data"),
+        ("is_updatable", "yes_or_no"),
+        ("is_insertable_into", "yes_or_no"),
+        ("is_trigger_updatable", "yes_or_no"),
+        ("is_trigger_deletable", "yes_or_no"),
+        ("is_trigger_insertable_into", "yes_or_no"),
+    ],
+}
+
+# retrived with:
```

Comment:
This doesn't need to go into this PR, but since this is all generated data, I think it might be worth having it be explicitly generated by a tool (along with `range_functions.py`)